include/plugin_api.inc.php

```diff
@@ -1308,9 +1308,9 @@ static function load_language($path) {
         $probelang = $path . '/' . $serendipity['charset'] . 'lang_' . $serendipity['lang'] . '.inc.php';
         if (file_exists($probelang)) {
             include $probelang;
+        } else {
```

Comment:
I think this can change the whole language flow quite a lot, if english language is not overloaded over a selected one?! IMO it should always fall back to english, because a translation might not contain new constants that have been added in the english file but not in the translation

Reply:
That's impossible now. The moment a constant is already defined redefining it will throw a warning,  @ or no @. We have to change how we did that.

But you are right, this is a breaking change.

Reply:
Sigh, ok. I haven't tried that specifically yet, so even if we (before the language include) do some error_reporting() redirection we can't just load the undefined constants from a file? It would be a pain to change all language files to something like "if (!defined('LANG_STUFF')) define('LANG_STUFF', 'Content')"...

Reply:
We in general don't need this. Given that everything renders it seems like we always redefine all the constants anyway, at least in the core and default plugins.

> so even if we (before the language include) do some error_reporting() redirection we can't just load the undefined constants from a file?

We might be able to hide all the Warnings for a moment with some voodoo, but that's magic to me. I couldn't build that hack, and think we shouldn't do it - would it work on all servers etc.?